src/main/java/org/example/Model/Account.java

```diff
@@ -1,4 +1,87 @@
 package org.example.Model;
 
+import jakarta.persistence.Entity;
+import jakarta.persistence.Id;
+import jakarta.persistence.Table;
+
+import java.util.UUID;
+
+@Entity
+@Table(name = "account")
 public class Account {
+
+    @Id
+    private UUID account_id;
+    private long customer_id;
+    private String currency;
+    private String iban;
+    private String bic;
+    private int balance;
+    private String status;
+    private String created_at;
+
+    public UUID getAccount_id() {
+        return account_id;
+    }
+
+    public void setAccount_id(UUID account_id) {
+        this.account_id = account_id;
+    }

```

Comment:
Good pattern that is widely user: https://medium.com/miro-engineering/fluent-setter-breaking-the-convention-33ce3433126e.

Just make it return itself and you can chain methods together.